### PR TITLE
Adding OpenAI Whisper encoder to ASR frontend

### PR DESCRIPTION
**Pull request by slSeanWU**
Hi all,

Here are some results I got with the config associated with this PR
- `Librispeech`

    | WER | dev-clean  | test-clean | dev-other | test-other |
    |:-----: |:-----: | :-----: |:-----: |:-----: |
    | Whisper medium (250M params) | 5.1 | 5.2 | 10.5 | 10.7 |
    | WavLM large (320M params), single-layer feat | 3.7 | 3.6 | 6.6 | 6.7 |
- `CHiME-4`

  | WER | dt05-1ch-real | dt05-1ch-simu | et05-1ch-real | et05-1ch-simu |
  |:-----: |:-----: | :-----: |:-----: |:-----: |
  | Whisper medium (250M params) | 7.3 | 10.6 | 10.8 | 14.0 |
  | WavLM large (320M params), single-layer feat | 7.7 | 9.4 | 7.9 | 11.9 |

### CODE REVIEW

File: tools/installers/install_whisper.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+if torch.cuda.is_available():
+    version=torch.version.cuda.split(".")
+    # 10.1.aa -> 101
+    print(version[0] + version[1])
+else:
+    print("")
+EOF
+)
+echo "cuda_version=${cuda_version}"
+
```
Comment (simpleoier) [inline tools/installers/install_whisper.sh:48]: Can you leave a note about the git commit hash info in case it breaks sometime in the future?

**Review by simpleoier — APPROVED** [review]
LGTM!

File: tools/installers/install_whisper.sh (hunk 1 above)
Comment (slSeanWU) [reply tools/installers/install_whisper.sh:48]: Ok, just add commit hash as a comment?

Comment (slSeanWU) [reply tools/installers/install_whisper.sh:48]: I think I'll just upload training config for Whisper frontend, as I already did

Comment (simpleoier) [reply tools/installers/install_whisper.sh:48]: yeah, just commit hash as a comment.
Sorry, I mean the results of this config on CHiME4 and Librispeech_100 which you are training now, for a record.

Comment (slSeanWU) [reply tools/installers/install_whisper.sh:48]: Oh, I see. I'll add them to the recipe READMEs. Does this sound good?

**Pull request closed without merging**